index.html

```diff
@@ -72,9 +69,23 @@ <h1 class='myname'>Lacey Vu</h1>
 
       <script src='js/jquery.js'></script>
       <script src='vendor/handlebars-v4.0.5.js'></script>
-      <script src='js/jobs.js'></script>
-      <script src='js/portfolio.js'></script>
-      <script src='js/app.js'></script>
+      <script src='vendor/marked.js'></script>
+      <script src='js/job.js'></script>
+      <script src='js/jobView.js'></script>
+      <script src='js/project.js'></script>
+      <script src='js/projectView.js'></script>
+      <script src='js/movie.js'></script>
+
+
+      <script>
```

Comment:
recommend an index.js to hold this code or add it to an existing script -- always best to get the .js out of your html when possible